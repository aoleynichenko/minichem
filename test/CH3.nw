start CH3
echo

geometry
C     0.08745162    -0.08744725    -0.08742186
H     0.52503428     0.78909888    -0.52508604
H    -0.78884450    -0.52530342    -0.52536318
H     0.52530257    -0.52529218     0.78892712
end

# STO-3G
basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
     0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

scf
  uhf
#  nodiis
#  direct
  doublet
end

task scf


