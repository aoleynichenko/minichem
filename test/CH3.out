Minichem 0.0
By Alexander Oleynichenko, 2016
Build:      1370
Build date: Mar 12 2016 16:13:31
Compiler:   gcc 5.2.0
Date:       Sat Mar 12 16:14:54 2016
Cores:      1
Arguments:  ./minichem CH3.nw 
------------------------------------------------------------
>>> CH3.nw
start CH3
echo

geometry
C     0.08745162    -0.08744725    -0.08742186
H     0.52503428     0.78909888    -0.52508604
H    -0.78884450    -0.52530342    -0.52536318
H     0.52530257    -0.52529218     0.78892712
end

# STO-3G
basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
     0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

scf
  uhf
  doublet
end

task scf


------------------------------------------------------------
                    Basis information
   H
  ----
  1   1S     3.42525091     0.15432897
             0.62391373     0.53532814
             0.16885540     0.44463454


   C
  ----
  1   1S    71.61683700     0.15432897
            13.04509600     0.53532814
             3.53051220     0.44463454

  2   2S     2.94124940    -0.09996723
             0.68348310     0.39951283
             0.22228990     0.70011547

  3   2P     2.94124940     0.15591627
             0.68348310     0.60768372
             0.22228990     0.39195739


Spherical basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   H     | ao basis        |   1    |     3      |
|   C     | ao basis        |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------
          ********************************
          *      Parallel SCF Module     *
          ********************************

                    SCF Parameters
                    --------------
               Wavefunction : UHF
             Max iterations : 50
               Density conv : 1e-06
                Energy conv : 1e-07
                       DIIS : on
                    diisbas : 5

Formula:   
Atoms:     4
Electrons: 9
Mol mass:  15.035
Charge:    0
Spin mult: 2
Units:     angstroms
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika
One-electron integrals done in 0.000001 sec
Basis orthogonalization algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.25052657
AO basis orthogonalization done in 0.000083 sec

Initial guess: extended Huckel theory
Initial energy = -48.56977360
Initial guess done in 0.000025 sec

#bfns = 8
#eris = 666

 iter.       Energy         Delta E       RMS-Dens       time
---------------------------------------------------------------
   1     -39.43862546    -0.59994813     0.22085043    0.03
                                         0.16942407
   2     -39.04517277     0.39345270     0.04503756    0.07
                                         0.04097303
   3     -39.07578728    -0.03061452     0.01544750    0.10
                                         0.01261844
   4     -39.07565891     0.00012838     0.00639313    0.13
                                         0.00604319
   5     -39.07620997    -0.00055106     0.00356202    0.16
                                         0.00348265
   6     -39.07640437    -0.00019441     0.00218267    0.20
                                         0.00215361
   7     -39.07650334    -0.00009897     0.00136883    0.23
                                         0.00135187
   8     -39.07655802    -0.00005468     0.00086255    0.26
                                         0.00085183
   9     -39.07659028    -0.00003227     0.00054413    0.30
                                         0.00053731
  10     -39.07660995    -0.00001967     0.00034336    0.33
                                         0.00033904
  11     -39.07662214    -0.00001219     0.00021669    0.36
                                         0.00021395
  12     -39.07662976    -0.00000762     0.00013675    0.39
                                         0.00013502
  13     -39.07663454    -0.00000478     0.00008630    0.43
                                         0.00008521
  14     -39.07663755    -0.00000301     0.00005447    0.46
                                         0.00005378
  15     -39.07663945    -0.00000189     0.00003437    0.49
                                         0.00003394
  16     -39.07664064    -0.00000119     0.00002169    0.52
                                         0.00002142
  17     -39.07664140    -0.00000075     0.00001369    0.56
                                         0.00001352
  18     -39.07664187    -0.00000048     0.00000864    0.59
                                         0.00000853
  19     -39.07664217    -0.00000030     0.00000545    0.62
                                         0.00000538
  20     -39.07664236    -0.00000019     0.00000344    0.65
                                         0.00000340
  21     -39.07664248    -0.00000012     0.00000217    0.69
                                         0.00000214
  22     -39.07664255    -0.00000008     0.00000137    0.72
                                         0.00000135
  23     -39.07664260    -0.00000005     0.00000086    0.75
                                         0.00000085
---------------------------------------------------------------
          Total SCF energy =   -39.07664260
  Nuclear repulsion energy =     9.73109626

Allocated: 194096 bytes
Freed:     194096 bytes
Max usage: 8960 bytes
Total time: 0.75 sec
